selfdrive/car/toyota/values.py

```diff
@@ -53,7 +53,7 @@ class CAR:
   LEXUS_IS = "LEXUS IS 2018"
   LEXUS_CTH = "LEXUS CT HYBRID 2018"
   RAV4H_TSS2 = "TOYOTA RAV4 HYBRID 2019"
-  LEXUS_NXH = "LEXUS NX HYBRID 2018"
+  LEXUS_NXH = "LEXUS NX HYBRID 2018 & 2019"
```

Comment:
Revert this change

Reply:
Just in the readme, the platform is the 2018 and up but it identifies as 2018. That's why all cars only have 1 year listed

Reply:
Understood. Thanks, Chris.